spec/bowling_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'bowling'
+
+describe Bowling do
```

Comment:
Clear testing on maximum score and example rolls. Could you also test for when a player scores a spare?